conf/documentation.conf

```diff
@@ -648,6 +648,46 @@ violations with an action of "email", or any other PacketFence-related
 message.
 EOT
 
+[alerting.encryption]
```

Comment:
Not quite sure about the configuration parameter name...
I'd say something like
alerting.smtp_encryption